chain-core/src/tx/fee/mod.rs

```diff
@@ -45,10 +45,27 @@ impl Milli {
     /// takes the integer part and 3-digit fractional part
     /// and returns the 3-digit fixed decimal number (i.fff)
     #[inline]
-    pub const fn new(i: u64, f: u64) -> Self {
+    pub(crate) const fn new(i: u64, f: u64) -> Self {
```

Comment:
Can we remove this function?

Reply:
because of the code [`const MAX_SLASH_RATIO: Milli = Milli::new(1, 0);`](https://github.com/crypto-com/chain/blob/master/chain-core/src/init/params.rs#L13), we need a const function, but we can not set the `try_new` function as const, if remove the `const fn new()`, we need `lazy_static` or `once_cell` to set the const value.